libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -574,6 +591,30 @@ void AP_InertialSensor_Backend::update_accel(uint8_t instance)
         _imu._accel_filter[instance].set_cutoff_frequency(_accel_raw_sample_rate(instance), _accel_filter_cutoff());
         _last_accel_filter_hz = _accel_filter_cutoff();
     }
+
+    // possily update the harmonic notch filter parameters
+	if (!is_equal(_last_harmonic_notch_bandwidth_hz, harmonic_notch_bandwidth_hz()) ||
+		!is_equal(_last_harmonic_notch_attenuation_dB, harmonic_notch_attenuation_dB()) ||
+		sensors_converging()) {
+		_imu._accel_harmonic_notch_filter[instance].init(_accel_raw_sample_rate(instance), harmonic_notch_center_freq_hz(), harmonic_notch_bandwidth_hz(), harmonic_notch_attenuation_dB());
```

Comment:
These settings are synchronized - I would just move the update into the gyro update and be done with this check